deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -500,13 +509,36 @@ export class StackComposer {
                 stage: stage,
                 defaultDeployId: defaultDeployId,
                 fargateCpuArch: fargateCpuArch,
+                targetGroups: [networkStack.albSourceProxyTG],
                 env: props.env
             })
             this.addDependentStacks(captureProxyStack, [elasticsearchStack, migrationStack,
                 kafkaBrokerStack, mskUtilityStack])
             this.stacks.push(captureProxyStack)
         }
 
+        let targetClusterProxyStack
+        if (targetClusterProxyServiceEnabled && networkStack && migrationStack) {
+            targetClusterProxyStack = new CaptureProxyStack(scope, "target-cluster-proxy", {
+                serviceName: "target-cluster-proxy",
+                vpc: networkStack.vpc,
+                customSourceClusterEndpointSSMParam: MigrationServiceCore.getStringParameterName(scope, SSMParameter.OS_CLUSTER_ENDPOINT, {stage, defaultDeployId}),
```

Comment:
Are you able to run this get SSM parameter command outside the context of a stack? Didn't think this was possible

Reply:
Modified for clarity